registrasion/forms.py

```diff
@@ -1319,11 +1320,14 @@ def contains_errors(self):
         required=False,
     )
 
-try:
-    VoucherForm = util.get_object_from_name(settings.VOUCHER_FORM)
-except:
-    VoucherForm = _VoucherForm
+def get_form_class_from_settings(config_name, default):
```

Comment:
might suggest moving this up to the top of the file for discoverability.

Reply:
Fixed in da25274b34fb2fd0f26bc0f1e44961ac6bf0fc38